RecoLocalCalo/EcalRecProducers/python/ecalMultiFitUncalibRecHit_cff.py

```diff
@@ -1,10 +1,17 @@
 import FWCore.ParameterSet.Config as cms
+from HeterogeneousCore.CUDACore.SwitchProducerCUDA import SwitchProducerCUDA
 from Configuration.ProcessModifiers.gpu_cff import gpu
 
 # ECAL multifit running on CPU
-from RecoLocalCalo.EcalRecProducers.ecalMultiFitUncalibRecHit_cfi import ecalMultiFitUncalibRecHit
+from RecoLocalCalo.EcalRecProducers.ecalMultiFitUncalibRecHit_cfi import ecalMultiFitUncalibRecHit as _ecalMultiFitUncalibRecHit
+ecalMultiFitUncalibRecHit = SwitchProducerCUDA(
+  cpu = _ecalMultiFitUncalibRecHit
```

Comment:
Possibly
```suggestion
  cpu = _ecalMultiFitUncalibRecHit.clone()
```

Reply:
Here and in a couple other places I did not `clone()` the module because the original code didn't:
```diff
- from RecoLocalCalo.EcalRecProducers.ecalMultiFitUncalibRecHit_cfi import ecalMultiFitUncalibRecHit
+ from RecoLocalCalo.EcalRecProducers.ecalMultiFitUncalibRecHit_cfi import ecalMultiFitUncalibRecHit as _ecalMultiFitUncalibRecHit
+ ecalMultiFitUncalibRecHit = SwitchProducerCUDA(
+   cpu = _ecalMultiFitUncalibRecHit
```

So, changes to `ecalMultiFitUncalibRecHit.cpu` would reflect changes to the original `ecalMultiFitUncalibRecHit`, as in the original code.

However I agree that people may find it confusing, and I have no objections to always making a `clone()`, as long as @cms-sw/ecal-dpg-l2 and @cms-sw/hcal-dpg-l2 and/or @cms-sw/reconstruction-l2 agree.

Reply:
I would find a clone the better solution.